src/containers/ArticlePage/TopicArticlePage/components/TopicArticleTaxonomy.jsx

```diff
@@ -267,6 +283,13 @@ class TopicArticleTaxonomy extends Component {
 
     return (
       <Fragment>
+        {showResourceType && (
+          <ResourceTypeSelect
+            availableResourceTypes={filteredResourceTypes}
+            resourceTypes={stagedResourceTypeChanges}
+            onChangeSelectedResource={this.onChangeSelectedResource}
+          />
+        )}
```

Comment:
Dette blei fjerna fordi ressurstype ikkje lenger gir meining for emneartikler. Så det kan tas bort.

Reply:
cool. 
I første omgang så måtte jeg bare få ting til å kjøre uten feil, var mye konflikter å ta hensyn til

Reply:
Denne kan fremedels oppdaters.